src/__tests__/integration.ts

```diff
@@ -159,6 +161,27 @@ describe('graphql server', () => {
       expect(orgResult.content.donationLink).toBe('https://donate.alliesofopenbeta.com')
       expect(orgResult.content.instagramLink).toBe('https://instagram.com/alliesofopenbeta')
       expect(orgResult.content.description).toBe('We are allies of OpenBeta!')
+
+      // eslint-disable-next-line
+      await new Promise(res => setTimeout(res, 1000))
+
+      const orgHistory = await changelogDataSource.getOrganizationChangeSets()
+      expect(orgHistory).toHaveLength(2)
+
+      // verify changes in most recent order
+      expect(orgHistory[0].operation).toEqual(OperationType.updateOrganization)
+      expect(orgHistory[1].operation).toEqual(OperationType.addOrganization)
+
+      // history is shown most recent first
+      const updateRecord = orgHistory[0].changes
+      expect(updateRecord[0].dbOp).toEqual('update')
+      expect(updateRecord[0].fullDocument._change?.historyId).toEqual(orgHistory[0]._id) // should point to current change
+      expect(updateRecord[0].fullDocument.displayName).toBe('Allies of Openbeta')
+
+      const createRecord = orgHistory[1].changes
+      expect(createRecord[0].dbOp).toEqual('insert')
+      expect(createRecord[0].fullDocument._change?.historyId).toEqual(orgHistory[1]._id) // should point to current change
```

Comment:
In `src/model/__tests__/AreaHistoryDataSource.ts` we check `prevHistoryId`, but in our earlier PR, you mentioned that we're no longer supporting this feature, so we didn't implement this for Organizations therefore there is no `prevHistoryId` to check for. Just wanted to confirm that this understanding is accurate!

Reply:
we can remove prevHistoryId